synapse/lib/storm.py

```diff
@@ -1946,6 +1947,138 @@ async def execStormCmd(self, runt, genr):
 
         await runt.printf('For detailed help on any command, use <cmd> --help')
 
+class MergeCmd(Cmd):
```

Comment:
We now how *two* merge commands; ``view.merge`` and this ``merge``. feels like a confusing namespace :(

Reply:
not really, one merges the entire view at once with no granularity options, and the other operates on nodes ( like most of our other non-namespaced commands ). Are you proposing any specific change?